### DISCUSSION TITLE
Fully Connected Node support

### DISCUSSION THREAD

**Discussion by jrlvdr** [onnx/onnx#3833] [Q&A] [answered]
Does ONNX support a Fully Connected operator, I did not see its supported operator's list. If so, how it converts the FC node to another Op type?

**Comment by zhenhuaw-me** [comment, marked as answer]
@janakiramreddy3 In ONNX, [Gemm](https://github.com/onnx/onnx/blob/main/docs/Operators.md#gemm) implements the _fully connected_ semantic.

Alternatively, [MatMul](https://github.com/onnx/onnx/blob/main/docs/Operators.md#MatMul) can also be used (which might require additional operators such as `Sum` for bias add).

You might be interested in this issue https://github.com/onnx/onnx/issues/24
